### ISSUE TITLE
Bug where the extension tries to resolve a user login with the path name of a document as its login info.

### ISSUE THREAD

**Issue by boltex** [microsoft/vscode-pull-request-github#3108]
- Extension version: 0.32.0
- VSCode Version: 1.62.2
- OS: Ubuntu 20.04.3 LTS

Steps to Reproduce:

As I am testing an extension that I'm developing, which has no relation with git, github nor vscode-pull-request-github extension, I open and update documents in a custom filesystem that my extebsion created and I **get this message about login in with a "user" called '+node:felix.20210715225557.1:' which is weirdly just the filepath+name of an unrelated document the debug extension host has opened.** :

(error text copy/pasted here that is the text in blue in the screenshot below)

```
Error: GraphQL error: Could not resolve to a User with the login of '+node:felix.20210715225557.1:'.
	at new e (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:140)
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:20957
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:26916
	at Set.forEach (<anonymous>)
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:26893
	at Map.forEach (<anonymous>)
	at t.broadcastQueries (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:26844)
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:28394
	at Object.next (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:26711)
	at ve (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:24580)
	at he (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25009)
	at te.value (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25549)
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:14269
	at Set.forEach (<anonymous>)
	at Object.next (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:14232)
	at ve (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:24580)
	at he (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25009)
	at te.value (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25549)
	at ve (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:24580)
	at he (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25009)
	at te.value (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25549)
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1080:10920
	at runMicrotasks (<anonymous>)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
Error: GraphQL error: Could not resolve to a User with the login of '+node:felix.20200609174345.1:'.
	at new e (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:140)
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:20957
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:26916
	at Set.forEach (<anonymous>)
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:26893
	at Map.forEach (<anonymous>)
	at t.broadcastQueries (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:26844)
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:28394
	at Object.next (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:26711)
	at ve (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:24580)
	at he (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25009)
	at te.value (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25549)
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:14269
	at Set.forEach (<anonymous>)
	at Object.next (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1078:14232)
	at ve (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:24580)
	at he (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25009)
	at te.value (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25549)
	at ve (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:24580)
	at he (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25009)
	at te.value (/home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1056:25549)
	at /home/felix/.vscode/extensions/github.vscode-pull-request-github-0.32.0/dist/extension.js:1080:10920
	at runMicrotasks (<anonymous>)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

Screenshot of what is happenning: source on the left, debug extension host on the right where you can see the breadcrumb of the opened 'leo body' file shows its filename: felix.20210715225557 
![Screenshot from 2021-11-16 19-18-03](https://user-images.githubusercontent.com/847716/142086605-d0f250ef-6481-45a8-86fd-37445237cf04.png)
.1

So basically it seems the vscode-pull-request-github extension tried to login to a github account to update itself but it used this opened document's filename from my extension being ran and debugged in a debug extension host... again, my extension (leointeg) does nothing with your extension. (it just displays documents from a custom filescheme)

Sorry to bring up such a random bug - i just felt compelled to post it here in case it lights up an idea in someone's head about a weird seemingly unrelated bug. 

Not a big bug, but by far the weirdest I have seen. ;)

Sorry for my english,

**Comment by alexr00** [comment]
I think this is coming from the support for showing a user hover for a @<username> strings. There's a `console.log` in there that probably shouldn't be there.

**Comment by rchiodo** [comment]
@alexr00 do you have any verification steps? Thanks.

**Comment by alexr00** [comment]
To verify:
1. Have the pre-release version of GHPRI installed
2. Open a folder with a GitHub remote
3. Open the dev tools
4. In a file that's part of the workspace, add `@this_is_not_a_real_user` (or any other fake username) in a comment in that file. 
5. Hover over the fake user name
6. Verify that a GraphQL error is not printed to the console.
